Android.md

```diff
@@ -16,4 +16,17 @@
 1.  [Udacity Nanodegree Material Design for Android Developers](https://www.udacity.com/course/material-design-for-android-developers--ud862)
 1.  [Udacity Nanodegree Android Wear Development](https://www.udacity.com/course/android-wear-development--ud875A)
 
+### Kotlin
+### Google has Kotlin as an official programming language for Android development in Google IO17 . 
+Kotlin is a statically typed programming language that runs on the Java virtual machine and also can be compiled to JavaScript source code or use the LLVM compiler infrastructure. 
+As of Android Studio 3.0 (October 2017) Kotlin is a fully supported programming language by Google on the Android Operating System[4], and is directly included in the Android Studio 3.0 IDE package as an alternative to the standard Java compiler. The Android Kotlin compiler lets the user choose between targeting Java 6- or Java 8-compatible bytecode.
+
+1.  [Kotlin Bootcamp for Programmers - byGoogle (To start learning with the Kotlin Language. )](https://in.udacity.com/course/kotlin-bootcamp-for-programmers--ud9011)
+1.  [Kotlin for Android Developers - To shift your already existing Java code to Kotlin](https://in.udacity.com/course/kotlin-for-android-developers--ud888)
+
+
+## UI/UX
+1.  [A curated list of awesome Android UI/UX libraries ](https://github.com/wasabeef/awesome-android-ui) #AwesomeUI.
```

Comment:
UI/UX in here isn't relevant. Raise a different pull request for UI resources so that we have a separate file for those.